docs/api.rst

```diff
@@ -2,6 +2,10 @@ API Documentation
 =================
 
 .. autosummary::
-   :toctree: autosummary
+    :toctree: stubs
```

Comment:
Just a suggestion, have you looked into the :recursive: option to see if that gets what you want with less work?

Reply:
I just checked this out. It doesn't display things quite how I want so I am going to stick with the current method. Will definitely keep `:recursive:` in mind if this package gets much bigger though. Thanks for the suggestion.